src/library/scala/collection/immutable/StringOps.scala

```diff
@@ -13,8 +13,8 @@ package immutable
 import mutable.StringBuilder
 
 /**
- *  This class serves as a wrapper providing `String`s with all the operations
- *  found in indexed sequences. Where needed, instances of `String` object
+ *  This class serves as a wrapper providing [[scala.Predef.String]]s with all the operations
+ *  found in indexed sequences. Where needed, instances of `String` objects
```

Comment:
The word objects doesn't actually belong. Even "instances of String" is possibly redundant. "Strings are implicitly converted into instances of this class."